plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -418,10 +418,69 @@ export const initProductScreenTracks = () => {
 				'.woocommerce_attribute'
 			).length;
 			if ( newAttributesCount > attributesCount ) {
+				const global_attributes = [
+					...document.querySelectorAll( '.pa_glbattr' ),
+				].map( ( globalAttr ) => {
+					const terms =
+						(
+							globalAttr.querySelector(
+								'.attribute_values'
+							) as HTMLSelectElement
+						 ).selectedOptions?.length ?? 0;
+
+					return {
+						visibleOnTheProductPage: (
+							globalAttr.querySelector(
+								'[name="attribute_visibility[1]"'
+							) as HTMLInputElement
+						 )?.checked
+							? 'yes'
+							: 'no',
+						usedForVariations: (
+							globalAttr.querySelector(
+								'[name="attribute_variation[1]"'
+							) as HTMLInputElement
+						 )?.checked
+							? 'yes'
+							: 'no',
+						terms,
+					};
+				} );
+				const local_attributes = [
+					...document.querySelectorAll(
+						'.woocommerce_attribute:not(.pa_glbattr)'
+					),
+				].map( ( attr ) => {
+					const terms =
+						(
+							attr.querySelector(
+								"[name='attribute_values[0]']"
+							) as HTMLTextAreaElement
+						 )?.value.split( '|' ).length ?? 0;
+					return {
+						visibleOnTheProductPage: (
+							attr.querySelector(
+								'[name="attribute_visibility[1]"'
+							) as HTMLInputElement
+						 )?.checked
+							? 'yes'
+							: 'no',
+						usedForVariations: (
+							attr.querySelector(
+								'[name="attribute_variation[1]"'
+							) as HTMLInputElement
+						 )?.checked
+							? 'yes'
+							: 'no',
+						terms,
+					};
+				} );
 				recordEvent( 'product_attributes_add', {
 					page: 'product',
 					enable_archive: '',
 					default_sort_order: '',
+					global_attributes,
```

Comment:
Does this apply?

Reply:
I'm not sure what you mean. It works, it's logging the global attributes "attached" to the product.

Reply:
Do you think this is worth it or should we keep a clear separation between both? 

Example: only track local attributes added on the add product screen, and track global attributes only at `wp-admin/edit.php?post_type=product&page=product_attributes` page

Reply:
> Do you think this is worth it or should we keep a clear separation between both?

This is what I meant (apologies for the lack of clarity): keep only relevant props and do not store redundant/repeated data. 
I'm a bit on the edge about including such cumulative information in events that mark an addition of an element. The event is not describing a product, but an action. What do you think about adding such information (name, slug, used for variations etc.) relevant to a new element for which the event was fired?